assessment.md

```diff
@@ -239,17 +239,39 @@ TODO: add description for each assessment level for this skill.
 ### Interactions
 
 * Interaction with the customers
+- SERVICE
+- Social
+- Eager
+- Replace
+- Verbal Communication
+- Informative
+- Countability
+- Easy
```

Comment:
I think the acronym you wrote down is great! However, you should still add an assessment scale similar to the other points below